CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+0.5.0 (2021-06-25)
+===
+
+GC Plans
+---
+* Added a new plan PageProtect, a plan to help debugging. It allocates each object
+  to a separate page, and protects pages when the pages are released.
+
+API
+---
+* Major changes to the ObjectModel trait: now a binding can specify each per-object
```

Comment:
... now a binding must specify...

I won't compile if the binding doesn't do this